components/home/sections/WhatCanYouDo.js

```diff
@@ -153,16 +147,13 @@ const WhatCanYouDo = () => {
           <FormattedMessage id="home.whatCanYouDoSection.manageMoney" defaultMessage="Manage Money" />
         </Title>
         <Box width={[null, '390px', '488px', null, '558px']} mr={[null, null, 3, null, 5]}>
-          <Illustration
+          <NextIllustration
+            width={558}
+            height={350}
             src="/static/images/home/managemoney-illustration-md.png"
```

Comment:
We should use the largest versions by default. NextJS will take care of the compression for us.

```suggestion
            src="/static/images/home/managemoney-illustration-lg.png"
```

I actually think we should take all our images for the homepage from `public/static/images/home/original`, to start from the highest quality possible.

```suggestion
            src="/static/images/home/original/managemoney-illustration-lg.png"
```

To take an informed decision, we need to properly check the size results of both of these approaches and see how they compare

Reply:
@Betree : I've corrected the above and also cleaned up some image files that are no longer used after this change. The testing indicates that there's no visual difference using the highest quality image. For the size with using the larger images (with this PR) is 339.2 Mb for the `.next` folder and without the PR it is, 390.4 MB. Interestingly the size is larger before probably because I removed a bunch of images. 😄 Let me know your thoughts.